content/academy/caching_responses_in_puppeteer.md

```diff
@@ -0,0 +1,169 @@
+---
+title: How to optimize Puppeteer by caching responses
+description: Learn why it's important to cache responses in memory when intercepting requests in Puppeteer, and how to do it.
+menuWeight: 19
+category: tutorials
+paths:
+    - caching-responses-in-puppeteer
+---
+
+# [](#caching-responses-in-puppeteer) Caching responses in Puppeteer
+
+> In the latest version of Puppeteer, the request-interception function inconveniently disables the native cache and significantly slows down the crawler. Therefore, it's not recommended to follow the examples shown in this article. Puppeteer now uses a native cache that should work well enough for most use cases.
```

Comment:
`Therefore, it's not recommended to follow the examples shown in this article unless you have a very specific use-case where the default browser cache is not enough (e.g. cashing over multiple scraper runs)`

Reply:
Otherwise, I would not add the article at all if we think it is useless :)